src/main/java/com/iota/iri/service/snapshot/impl/LocalSnapshotManagerImpl.java

```diff
@@ -34,7 +34,7 @@
     /**
      * Service that contains the logic for generating local {@link com.iota.iri.service.snapshot.Snapshot}s.
      */
-    private static final LocalSnapshotService localSnapshotService = new LocalSnapshotServiceImpl();
+    private static final SnapshotService SNAPSHOT_SERVICE = new SnapshotServiceImpl();
```

Comment:
Currently it doesn't matter that this is static since it doesn't have any dependencies.
Usually services should have other services/repositories(data providers) as dependencies and then it can't be static (just a note).

Also usually it should be a non-static field that you should be obtain via an injection from `IOTA.class`. The reason is that maybe more than one module will need the service. For example maybe `LedgerValidator` will need sometimes in the future logic from `SnapshotService`.

Question:
Did codacy tell you to capitalize it? The convention is when you `static final primitive` then always capitalize. When you have `static final Object` java convention is unclear. Some projects capitalize and some don't. In this project up until now we didn't capitalize. Wondering why you chose to capitalize now.

Reply:
i changed it back already - it is now a parameter because the SnapshotProvider also needs it. So it will be used like the rest of the Dependency Injection stuff.